.config-schema.json

```diff
@@ -184,6 +184,12 @@
                     "type": "string",
                     "enum": ["text", "json"],
                     "default": "text"
+                },
+                "level": {
+                    "description": "The log level to set. For production we recommend 'info' format.",
```

Comment:
@craigpastro would you please submit a PR to openfga.dev and add this to the Production Checklist?

Reply:
Sure. Thanks!